firebase-sessions/test-app/test-app.gradle.kts

```diff
@@ -45,6 +45,24 @@ android {
   }
   kotlinOptions { jvmTarget = "1.8" }
   buildFeatures { viewBinding = true }
+  buildTypes {
+    release {
+      // We only want to actually crash the app for the scheduled runs, not the integration tests
+      buildConfigField(
+        "boolean",
+        "SHOULD_CRASH_APP",
+        if (project.hasProperty("useReleasedVersions")) "true" else "false"
```

Comment:
This should already be a non-null boolean. Can you call toString or something?

Reply:
Done